index.js

```diff
@@ -59,15 +59,25 @@
           reject('No year end found.')
         }
       }).catch(function(err) {
-          reject('Problem with reading file', err);
+          reject('Problem with reading file\n' + err);
       })
     })
 
     // Utility functions
     function loadField(conceptToFind, fieldName, key) {
       key = key || '$t';
       fieldName = fieldName || conceptToFind;
-      self.fields[fieldName] = _.get(self.documentJson, ['dei:' + conceptToFind, key], 'Field not found.');
+      var concept = _.get(self.documentJson, 'dei:' + conceptToFind);
+      //console.log(fieldName + "=> " + JSON.stringify(concept, null, 3));
+      if(_.isArray(concept)) {
+        // Default to the first available contextRef
+        console.warn('Found ' + concept.length + ' context references')
+        _.forEach(concept, function(c, idx) { console.warn('=> ' + c.contextRef + (idx === 0 ? ' (selected)' : ''))});
```

Comment:
Please spread this logic out over multiple lines, and use a longer variable name than c.